### DISCUSSION TITLE
在交叉引用里面找不到图的引用类型

### DISCUSSION THREAD

**Discussion by Huoyucat** [sikouhjw/gdutthesis-Word#8] [Q&A] [answered]
在交叉引用里面找不到图的引用类型，除了编号项以外的类型都找不到

**Comment by sikouhjw** [comment, marked as answer]
参考《交叉引用》[^1]一节，『引用类型』为『图』。

[^1]: https://github.com/sikouhjw/gdutthesis-Word/wiki#%E4%BA%A4%E5%8F%89%E5%BC%95%E7%94%A8

----------

如果找不到需要的引用类型，目前想到的解决方案：『引用』->『题注』->『插入题注』->『新建标签』，将没有的标签手动添加一下，然后就可以引用了。

如何知道标签的名称？`Alt`+`F9` 可以看到域代码，例如 `{ SEQ 图 \* ARABIC \s 1 }`，这里的名称就是 `图`。

**Reply by Huoyucat** [reply]
![20220314 210955746](https://user-images.githubusercontent.com/74894658/158178430-564b94fd-754c-443f-becd-2b94ebe8e9d3.png)
如图显示，figure里面没有标签

**Reply by sikouhjw** [reply]
> 如图显示，figure里面没有标签

是 `图`，不是 `Figure`，我用别人电脑的 WPS 测试，可以找到需要的引用类型。

如果没有 `图`，请提供下（将关键信息删减后的）文件。

**Reply by Huoyucat** [reply]
[本科毕业设计o11.docx](https://github.com/sikouhjw/gdutthesis-Word/files/8245194/o11.docx)
这个我单纯就是另存为了一下，看一下谢谢

**Reply by sikouhjw** [reply]
> 这个我单纯就是另存为了一下，看一下谢谢

我的电脑上是没有问题的，我初步怀疑『题注』里面的引用类型是保存在本地、而不随着模板分发的。

目前想到的解决方案：『引用』->『题注』->『插入题注』->『新建标签』，将没有的标签手动添加一下，然后就可以引用了。

如何知道标签的名称？`Alt`+`F9` 可以看到域代码，例如 `{ SEQ 图 \* ARABIC \s 1 }`，这里的名称就是 `图`。

**Reply by Huoyucat** [reply]
新建标签之后就可以了，感谢大哥😀

**Reply by sikouhjw** [reply]
> 新建标签之后就可以了，感谢大哥😀

我后续测试一下是谁的问题，好像自动图文集也会出现“消失”的问题，不清楚是不是 Word 的缺陷。

如果测试出是 Word 自身的缺陷，我会在 Wiki 里补充上。

**Reply by Huoyucat** [reply]
自动图文集也会“消失”我在word2016遇到过，更新到2021就没这个问题

**Reply by sikouhjw** [reply]
经测试，是 Word 不能分发用户新建的标签，目前的解决方案是：在 Wiki 里提示如何新建标签，因为 Word 自带的标签是不够用的，必须新建标签才够用。
